medusa/search/proper.py

```diff
@@ -66,6 +67,8 @@ def run(self, force=False):  # pylint: disable=unused-argument
         if force:
```

Comment:
So it will only apply the custom_search_days, when the 'force' applies? Meaning, you will need to forcibly search for a proper? That kind of defeats the purpose, as you never know when you need to search for a proper right? Also as i'm using nzb, propers can arrive later then at a good torrent provider. Meaning, I might want to extend the default search period for propers?